custom_resources/elasticsearch.py

```diff
@@ -0,0 +1,33 @@
+"""Custom resources related to Elasticsearch."""
+from six import string_types
+from .LambdaBackedCustomResource import LambdaBackedCustomResource
+
+
+class IngestPipelineViaVpc(LambdaBackedCustomResource):
+    props = {
+        'EsHost': (string_types, True),
+        'PipelineName': (string_types, True),
+        'IngestDocument': (dict, True),
+    }
+
+    @classmethod
+    def _lambda_policy(cls):
+        return {
+            "Version": "2012-10-17",
+            "Statement": [{
+                "Effect": "Allow",
+                "Action": "es:ESHttp*",
```

Comment:
I think we can scope this down more (definitely for action, possibly for resource)

Reply:
I added commit: [cf0b80f](https://github.com/vrtdev/custom-resources/pull/46/commits/cf0b80fcfa718bbf227c4370f4fc63d0ebebdefd) which only allows ESHttpPut and ESHttpDelete